### PR TITLE
chatroom, reminder, and notification automation

### PR DESCRIPTION
**Pull request by priagungs**

### CODE REVIEW

File: src/main/java/com/future/function/qa/util/Cleaner.java [hunk 1]
```diff
@@ -61,4 +64,9 @@ private void setMarkUsedFalse(String collectionName, String id) {
       .updateOne(new BasicDBObject("_id", id), Updates.set("used", false));
   }
 
+  public void hardDeleteMessageStatus(String messageId) {
```
Comment (jonathan016) [inline src/main/java/com/future/function/qa/util/Cleaner.java]: Set private

File: src/main/java/com/future/function/qa/util/Cleaner.java [hunk 2]
```diff
@@ -61,4 +64,9 @@ private void setMarkUsedFalse(String collectionName, String id) {
...
+    mongoDatabase.getCollection(DocumentName.MESSAGE_STATUS)
+            .deleteMany(new BasicDBObject("message.$id", ObjectId.isValid(messageId) ? new ObjectId(messageId) : messageId));
```
Comment (jonathan016) [inline src/main/java/com/future/function/qa/util/Cleaner.java:69]: Sorry, what is the business flow here? Why delete message status by message id?

File: src/test/java/com/future/function/qa/DefinitionTestSuite.java [hunk 3]
```diff
@@ -6,5 +6,5 @@
 import net.serenitybdd.cucumber.CucumberWithSerenity;
 
 @RunWith(CucumberWithSerenity.class)
-@CucumberOptions(features = "src/test/resources/features/", tags = "@Regression", junit = "--step-notifications")
+@CucumberOptions(features = "src/test/resources/features/", tags = "@Reminder", junit = "--step-notifications")
```
Comment (jonathan016) [inline src/test/java/com/future/function/qa/DefinitionTestSuite.java]: Revert to `@Regression`

File: src/test/java/com/future/function/qa/steps/Scoring/Question/QuestionSteps.java [hunk 4]
```diff
@@ -194,4 +194,5 @@ public void questionBaseResponseCodeShouldBe(int expectedCode) {
   public void questionBaseResponseStatusShouldBe(String expectedStatus) throws Throwable {
     assertEquals(expectedStatus, questionData.getBaseResponse().getStatus());
   }
+
```
Comment (jonathan016) [inline src/test/java/com/future/function/qa/steps/Scoring/Question/QuestionSteps.java]: Revert this

**Review by jonathan016 — CHANGES_REQUESTED** [review]

File: src/main/java/com/future/function/qa/util/Cleaner.java (hunk 2 above)
Comment (priagungs) [reply src/main/java/com/future/function/qa/util/Cleaner.java:69]: For every create message api called, some message status also will be created. I cant get the message status id related to that message, so i need to remove the message status by using message id.

Comment (jonathan016) [reply src/main/java/com/future/function/qa/util/Cleaner.java:69]: Ok, but a message cannot be deleted by user, right?

Comment (priagungs) [reply src/main/java/com/future/function/qa/util/Cleaner.java:69]: yep

File: src/test/java/com/future/function/qa/steps/Scoring/Question/QuestionSteps.java (hunk 4 above)
Comment (priagungs) [reply src/test/java/com/future/function/qa/steps/Scoring/Question/QuestionSteps.java]: done

File: src/test/java/com/future/function/qa/DefinitionTestSuite.java (hunk 3 above)
Comment (priagungs) [reply src/test/java/com/future/function/qa/DefinitionTestSuite.java]: done

File: src/main/java/com/future/function/qa/util/Cleaner.java (hunk 1 above)
Comment (priagungs) [reply src/main/java/com/future/function/qa/util/Cleaner.java]: done

**Review by jonathan016 — APPROVED** [review]

**Pull request merged**